### ISSUE TITLE
In production do not run the `remote` project from development servers

### ISSUE THREAD

**Issue by wurmr** [milesburton/teslacam#10]
Consider using `node` instead of `nodemon` to host the backend and host the compiled frontend either from the same node server or use `http-server` with the proxy switch.

This will make it so the slow raspberry pi hardware doesn't have to do the heavy lifting of webpack compile.

**Comment by milesburton** [comment]
Agreed, feel free to raise a PR. This is low on my list whilst I get to the bottom of why Tesla sometimes rejects the mounted USB

**Comment by wurmr** [comment]
Is there an open issue for that rejection? I _may_ have some insight on that. My USB splitter was hit and miss but plugging directly in it seems fine. `dmesg` output thru the splitter would show that the Tesla wouldn't come up as a device at all (even tho the Pi was getting power).

**Comment by milesburton** [comment]
> Is there an open issue for that rejection? I _may_ have some insight on that. My USB splitter was hit and miss but plugging directly in it seems fine. `dmesg` output thru the splitter would show that the Tesla wouldn't come up as a device at all (even tho the Pi was getting power).

Not yet, this GH has only started to gain traction outside of my own personal use. I've been waiting for a spare few minutes to sit in the back of the car with my laptop.

Originally it was an issue with partition tables, fixed that. Then it appeared to be the SDHC speed, fixed that. I've checked brownouts, that's also fixed.

Now I'm not entirely sure. I was debating whether to take a more powerful machine and mount a ramdisk and see how the car coped with it.

In short, would be good to get some more eyes on this code to get it back up and running again.
